securedrop/source_templates/index.html

```diff
@@ -54,12 +76,14 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can disable Javascript in 3 easy steps!</p> 
+      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
       <ol>
-        <li>Click the <img src="static/i/no16.png" alt="NoScript icon" /> NoScript icon in the toolbar above</li>
-        <li>Click <strong><img src="static/i/no16-global.png" alt="Forbid Scripts Globally" /> Forbid Scripts Globally (advised)</strong></li>
-        <li>If the page does not refresh automatically, <a href="/">click here</a> to refresh the page</li> 
+        <li>Click the <img src="{{ url_for('static', filename='i/toronion.png') }}" alt="Tor icon" />Tor icon in the toolbar above</li>
+        <li>Click <strong>Security Settings</strong></li>
+        <li>Turn the Slider to <strong>High</strong></li>
+        <li>If the page does not refresh automatically, <a href="/">click here</a> to refresh the page</li>
       </ol>
+
       <p><a href="/howto-disable-js">Not using the Tor Browser Bundle?</a></p>
```

Comment:
@garrettr Good point - modified this in 2905b42 and added the new template to the AppArmor rules in 5fe7d40 

@ninavizz - we're trying to just make final changes to clean up the code and push this release out ASAP so this isn't going to make it into this release - note that your comments are solid and we should totally fold these into the next release. If we get a PR targeting `develop` then we can get these additional changes in 💯